src/main/java/org/prebid/server/bidder/huaweiads/model/response/CreativeType.java

```diff
@@ -0,0 +1,34 @@
+package org.prebid.server.bidder.huaweiads.model.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+@Getter
+@AllArgsConstructor
```

Comment:
Better create explicit constructor. We tend to avoid these on enums. Also, please, check for similar occurences.

Reply:
what is the reason behind that?